src/index.js

```diff
@@ -137,9 +137,9 @@ module.exports = class Bitwap {
       cb(err, block)
     }
 
-    this.getBlocks([key], (err, res) => {
-      if (err) {
-        return done(err)
+    this.getBlocks([key], (errs, res) => {
+      if (errs) {
+        return done(errs[0])
```

Comment:
Maybe do like the `block-service`? Where getBlocks returns an object of `mutlihash: { err, block} `